### PR TITLE
Add support for backups

### PR DESCRIPTION
**Pull request by jvanbaarsen**
This is done in combination with the Backup gem, that is installed in the
Intercity-Docker project. There we will schedule the backups using Whatever to
cron.

I tested the working of this on one of our servers.

/cc @brambokdam

### CODE REVIEW

File: config/backup_schedule.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+every :day, :at => '1:00am' do
```
Comment (brambokdam) [inline config/backup_schedule.rb]: Maybe we can use the new syntax here, what do you think? `every :day, at: '1:00am' do`

Comment (jvanbaarsen) [inline config/backup_schedule.rb]: :+1:

**Comment by jvanbaarsen** [comment]
@brambokdam Can you do a re-review? some stuff has changed

File: config/backup/models/backup.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+Backup::Model.new(:intercity, 'Intercity full backup') do
+  ##
+  # PostgreSQL [Database]
+  #
+  database PostgreSQL do |db|
+    # To dump all databases, set `db.name = :all` (or leave blank)
+    db.name               = "intercity"
+    db.host               = "localhost"
```
Comment (brambokdam) [inline config/backup/models/backup.rb]: The production database can be hosted on another host, is this something that can cause issues?

Comment (jvanbaarsen) [inline config/backup/models/backup.rb]: I assume you have used the default intercity-docker system. Which assumes you host on the same machine. If not, you're on your own for now. (Agreed?)

**Pull request merged**